includes/setup.inc.sh

```diff
@@ -1,9 +1,27 @@
 #!/usr/bin/env bash
-set -x
 
-if ! command -v mvn3 >/dev/null; then
-  alias mvn3='mvn '
-  function mvn3() {
-      mvn "${@}"
-  }
+function __debug() {
+  # echo ${@}
+  true
+}
+
+if type brew &>/dev/null; then
```

Comment:
what if the user has an outdates maven 3.2.3 on brew because he installed it ages ago?

Reply:
They are potentially out of luck (depends on which demo they want to run). I will note in the readme that you need Maven 3.9.x installed.

For me, the primary goal of this repo is that we can run the demos in a reliable way. Attendees or users being able to run them on their machine is a secondary goal, so we help them to some extend, but we can't do everything for them.